app/Helpers/Helper.php

```diff
@@ -1458,5 +1458,9 @@ public static function mapBackToLegacyLocale($new_locale = null)
         }
         return $new_locale; // better that you have some weird locale that doesn't fit into our mappings anywhere than 'void'
     }
+    public static function sumThreshold($min_qty = null){
+
+        return $min_qty + Setting::getSettings()->alert_threshold;
+    }
```

Comment:
I have a hunch this method name can be improved but I don't have a good suggestion...